tests/s3_bucket/test_github_content_compare_slugs_styles.py

```diff
@@ -0,0 +1,107 @@
+import requests
+
+import urllib.parse
+from urllib.error import HTTPError
+
+import pytest
+
+from bs4 import BeautifulSoup
+
+
+"""
+Compares slugs/styles between books.xml and ABL file of every collection in github content repo.
+Latest update on June 6th, 2022
+"""
+
+
+def test_github_content_compare_slugs_styles(
+    git_content_repos, headers_data, abl_books_slugs_styles
+):
+
+    for repo in git_content_repos:
+
+        print("\nNow verifying: ", repo)
+
+        meta_inf_dir = f"https://api.github.com/repos/openstax/{repo}/contents/META-INF/"
```

Comment:
Keep in mind that the version of content on `main` may not match the version that has been added to the ABL. If that is important, you can include the `commit_sha` from the ABL in `abl_books_slugs_styles` and then use that with the `ref` query argument like 
```python
meta_inf_dir = f"https://api.github.com/repos/openstax/{repo}/contents/META-INF?ref={commit_sha}"
```

The way you are running this test right now would check if the `META-INF/books.xml` on `main` contains information that also exists anywhere in the ABL.

Reply:
@TylerZeroMaster hm... I am confused by this... how books.xml and ABL relate to each other? Should they match on main?

Reply:
I gave this more thought and it might not be a concern. The way it is now, you would only ever test the latest edition that has not yet been added to the ABL; however, I am not sure why you would need to test older editions.

The only other concern is that the contents of META-INF/books.xml on main may contain a style/slug that is not contained in the ABL yet. Granted, it might never be a problem.